tests/e2e/update-tag-profil.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { prepareNock, mockCredentials } from '../helpers'
+import CoinpaymentsClient from '../../src'
+
+import { CMDS } from '../../src/constants'
+
+describe('Update tag profile e2e test', () => {
+  let client
+  const VALID_API_PAYLOAD = {
+    tagid: 'tagid',
+  }
+  beforeAll(() => {
+    client = new CoinpaymentsClient(mockCredentials)
+  })
+  it('Should catch valid payload', async () => {
+    const VALID_PAYLOAD_MOCK = {
+      cmd: CMDS.UPDATE_TAG,
+      ...VALID_API_PAYLOAD,
+    }
+    const scope = prepareNock(mockCredentials, VALID_PAYLOAD_MOCK)
+    await client.updateTagProfile(VALID_API_PAYLOAD)
+    expect(scope.isDone()).toBeTruthy()
+  })
+  it('Should throw error on invalid payload', async () => {
+    for (const key in VALID_API_PAYLOAD) {

```

Comment:
to get proper error message should this fail, you might want to put iteration outside of `it()` block and include key in testcase name